src/internal/grpcutil/addr_test.go

```diff
@@ -17,15 +17,6 @@ func TestParsePachdAddress(t *testing.T) {
 	_, err = ParsePachdAddress("grpc://user:pass@pachyderm.com:80")
 	require.YesError(t, err)
 
-	_, err = ParsePachdAddress("grpc://pachyderm.com:80/")
```

Comment:
They no longer pass because I removed the "has no path" constraint. On second thought maybe I'll re-add that constraint when the protocol isn't `unix`.